test/random.spec.ts

```diff
@@ -304,6 +304,82 @@ describe('random', () => {
     });
   });
 
+  describe('numeric', () => {
+    it('should return single digit when no length provided', () => {
+      const actual = faker.random.numeric();
+
+      expect(actual).toHaveLength(1);
+      expect(actual).toMatch(/^[1-9]$/);
+    });
+
+    it.each(times(100))(
+      'should generate random value with a length of %s',
+      (length) => {
+        const actual = faker.random.numeric(length);
+
+        expect(actual).toHaveLength(length);
+        expect(actual).toMatch(/^[1-9]/);
+        expect(actual).toMatch(/[0-9]+$/);
```

Comment:
You should merge these checks:

```suggestion
        expect(actual).toMatch(/^[1-9][0-9]+$/);
```

Reply:
How does this work with a given `length = 1`?

Reply:
Use * instead of +